apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -149,7 +151,12 @@ describe('/research-outputs/ route', () => {
     });
 
     test('Should return 403 when user is not permitted to create research output', async () => {
-      const researchOutput = getResearchOutputPostRequest();
+      const {
+        // @ts-ignore
+        addedDate: _addedDate,
+        ...researchOutput
+      } = getResearchOutputPostRequest();
```

Comment:
if we are removing addedDate from the `ResearchOutputPostRequest` this fixture should not really contain an addedDate, and its probably what typescript was telling you until you added the ignore. Update the fixture to not contain the no longer supported property value